static/js/compose.js

```diff
@@ -787,7 +787,7 @@ function validate_stream_message() {
     }
 
     var current_stream = stream_data.get_sub(stream_name);
-    var stream_count = current_stream.subscribers.num_items();
+    var stream_count = (current_stream ? current_stream.subscribers.num_items() : 0);
```

Comment:
https://github.com/sampritipanda/zulip/blob/d806cfb32563227af3b1b5f1aff2933e680da876/static/js/compose.js#L813
In this line, it makes a request to the server to check if the stream actually exists and somehow the client doesn't know about it. If it doesn't, it shows the error you can see in the screenshot earlier.

Another option would be to move the @all thing after the stream checking is done with the server.